content/reference/important-protocols.adoc

```diff
@@ -40,49 +44,61 @@ reference] for details of their behaviors.
 These protocols break up the lifecycle of routes and allow both
 extension points and alternative strategies in several places.
 
-To give you an idea what that means, there are three built-in route
-definition syntaxes: table, terse, and verbose. There are three built
-in routing algorithms: linear-search, prefix-tree, and map-tree. An
-application can use any syntax with any algorithm. Or, an application
-can create a new syntax while using a built-in algorithm. Or, an
-application could use a built-in syntax with a new algorithm!
+Over time, Pedestal has evolved several data structures for representing routing;
+at the core, they map an HTTP method and URI path to a set of interceptors and a handler.
```

Comment:
May be worth mentioning that these data structures are tightly coupled to HTTP. Namespaces should suggest it, but may bear repeating.